policyengine_us/variables/gov/states/ms/tax/income/ms_taxable_income.py

```diff
@@ -7,4 +7,11 @@ class ms_taxable_income(Variable):
     label = "Mississippi taxable income"
     unit = USD
     definition_period = YEAR
+    reference = (
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=13",
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80105228.pdf",  # Line 38 - 49,
+    )
     defined_for = StateCode.MS
+
+    adds = ["ms_agi"]
+    subtracts = ["ms_deductions", "ms_total_exemptions"]
```

Comment:
can this be negative? usually we do a max_(0, ...)

Reply:
yes, this should be negative